conflowgen/tests/notebooks/compare_container_dwell_time_distribution_with_results.ipynb

```diff
@@ -69,13 +92,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
```

Comment:
Please never commit Jupyter Notebook files with output. They should be commited empty. Please reset them before committing them.

Reply:
See [Undo Jupyter notebook changes](https://github.com/1kastner/conflowgen/pull/187/commits/59b20657b4385cda8c81dffb046ece921603955a)